controllers/registry.go

```diff
@@ -761,15 +761,19 @@ func (r *DPAReconciler) parseAzureSecret(secret corev1.Secret, secretKey string)
 	return azcreds, nil
 }
 
-func (r *DPAReconciler) getMatchedKeyValue(matchedKey string, line string) (string, error) {
-	cleanedLine := strings.ReplaceAll(line, " ", "")
-	cleanedLine = strings.ReplaceAll(cleanedLine, "\"", "")
-	matchedKeyValue := strings.Replace(cleanedLine, matchedKey, "", -1)
-	if len(matchedKeyValue) == 0 {
-		r.Log.Info("Could not parse secret for %s", matchedKey)
-		return matchedKeyValue, errors.New("azure secret parsing error")
+// Return value to the right of = sign
+func (r *DPAReconciler) getMatchedKeyValue(key string, s string) (string, error) {
```

Comment:
@savitharaghunathan could you double check if changes here is ok for Azure.

Reply:
oh wait.. I found a problem..
https://github.com/openshift/oadp-operator/blob/06308a61b1f89d169a1c172626a130799821cc54/controllers/registry.go#L724
brb with fix.

Reply:
fixed in a5d575966e150e9843774565e9400b81c13418da